util/kube/portforwarder.go

```diff
@@ -71,6 +73,18 @@ func PortForward(targetPort int, namespace string, overrides *clientcmd.ConfigOv
 	}
 	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport}, "POST", url)
 
+	// Reuse environment variable for kubectl to disable the feature flag, default is enabled.
+	if !cmdutil.PortForwardWebsockets.IsDisabled() {
```

Comment:
Documenting that KUBECTL_PORT_FORWARD_WEBSOCKETS=false will not execute this logic